source/onboard/advanced-permissions-backend-infrastructure.rst

```diff
@@ -421,6 +421,13 @@ The following built-in roles and associated permissions are available:
 - sysconsole_read_compliance
 - create_emojis
 
+*system_read_only_admin*
+
+- create
+- edit
+- delete
+- manage members
+
```

Comment:
Did you mean to this?

```suggestion
*system_custom_group_admin*

- create_custom_group
- edit_custom_group
- delete_custom_group
- manage_custom_group_members

```

Reply:
That was a bad copy-pasta. Fixed! Thank you!